reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
Here too, ComposedShader and it parts fields are coming from X3D, and not X3DOM:
http://www.web3d.org/documents/specifications/19775-1/V3.3/Part01/components/shaders.html#ComposedShader

However, we have indeed the choice to rename `fields` to `uniforms` or `uniformVariables`, minus the complexity to rename it in the code when exporting to X3DOM.
I propose to rename it to `uniforms`.